### ISSUE TITLE
sample rate for EN_GB

### ISSUE THREAD

**Issue by faloupy** [aws-samples/amazon-transcribe-websocket-static#1]
Firstly, thanks to Karan for such a beautiful demo. I look forward to wading through the code in more detail later!

This isn't Karan's fault, the AWS docs state throughout that 16000Hz is allowed for EN_GB, but I have found that it accepts only 8000Hz for that ISO code.

It's a very small point, and when the transcribe team update the capabilities for EN_GB to match the docs, this will go away, but it might be worth mentioning that for anyone who plans to implement this wonderful tutorial, they should expect an error from the WS if using EN_GB and a sample rate of more than 8000 at this time. However EN_US does an amazing job of recognising EN_GB speech.

```
function setLanguage() {
    languageCode = $('#language').find(':selected').val();
    if (languageCode == "en-US" || languageCode == "es-US")
        sampleRate = 44100;
    else if(languageCode == "en-GB")
        sampleRate = 8000; 
    else
        sampleRate = 16000;
}
```

**Comment by brandonmwest** [comment]
Hi @faloupy, glad you are enjoying the demo. I'll pass the bug report over to the Transcribe team, as I believe it should downsample things on the server side.

**Comment by brandonmwest** [comment]
Closed by #4
